stl/inc/yvals_core.h

```diff
@@ -1262,7 +1262,7 @@
 #define __cpp_lib_erase_if                202002L
 
 #if _HAS_CXX23 && defined(__cpp_lib_concepts) // TRANSITION, GH-395 and GH-1814
-#define __cpp_lib_format 201907L
+#define __cpp_lib_format 202110L

```

Comment:
Gack, my mistake - we can't bump this until we implement #1981 and #2239. (Please revert the feature-test macro test changes accordingly.)